test/libevmcore/CMakeLists.txt

```diff
@@ -1,5 +0,0 @@
-cmake_policy(SET CMP0015 NEW)
-
-aux_source_directory(. SRCS)
-
-add_sources(${SRCS})
```

Comment:
Ok I see, you move the file in the libsolidity folder. But you could still leave CMakeLists.txt as it is. So one can add test files there without touching the CMake file.